### DISCUSSION TITLE
Troubleshooting Hubs Cloud Rollback Complete: SOLVED Disable Transfer Lock on domains

### DISCUSSION THREAD

**Discussion by cmloegcmluin** [mozilla/hubs#2751] [General Questions & Discussion] [answered]
**TL;DR: I fixed a failure in stack creation on AWS for Hubs Cloud Personal by disabling "Transfer lock" for my `.link` domain.**

If your stack is failing to create as well, try checking https://console.aws.amazon.com/route53/home#DomainListing: and ensuring that the "Transfer Lock" column for both of your domains is not checked. It should be an 'x'. If you've got a check, click your domain's name to go to its edit page, and then in the top center click "disable" next to "Transfer lock". Then just retry creating your stack.

Best of luck and let me know if you have any other questions.

**Comment by robinkwilson** [comment, marked as answer]
Great write up! I'll be integrating this into the documentation! 

https://github.com/MozillaReality/hubs-docs/issues/56
